mitosheet/src/mito/components/footer/GraphTabContextMenu.tsx

```diff
@@ -79,9 +81,10 @@ export default function GraphTabContextMenu(props: {
         props.setUIState(prevUIState => {
             return {
                 ...prevUIState,
+                selectedTabType: 'graph',
                 currOpenTaskpane: {
                     type: TaskpaneType.GRAPH, 
-                    graphSidebarTab: GraphSidebarTab.Export,
+                    graphSidebarOpen: true,
```

Comment:
Nice. This is fine for now. In the future, we should do the refactor of moving the graph to being a type of tab instead a type of taskpane, and then we could use our taskpane infrastructure to handle this. But we did a lot of graph code cleanup already so this is fine.

Reply:
Defaulting this to `true` when you are creating the graph for the first time is perfect. But if you are reopening a graph that already exists, let's default this to false.